src/Authentication/Controllers/SystemUserController.cs

```diff
@@ -124,14 +124,14 @@ public async Task<ActionResult> SetDeleteFlagOnSystemUser(Guid systemUserId)
         /// to ensure that there is no mismatch if the same partyId creates several new SystemUsers at the same time
         /// </summary>
         /// <returns></returns>    
-        [Authorize(Policy = AuthzConstants.POLICY_ACCESS_MANAGEMENT_WRITE)]
+        [Authorize(Policy = AuthzConstants.POLICY_ACCESS_MANAGEMENT_WRITE)]        
```

Comment:
what about the read policies? I see there aswell we have different parameter names

Reply:
changed